app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -0,0 +1,185 @@
+import co.nimblehq.compose.crypto.ui.common.price.PriceChangeButton
+import co.nimblehq.compose.crypto.ui.theme.Color
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp0
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp16
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp60
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp8
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp9
+import co.nimblehq.compose.crypto.ui.theme.Style
+import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+import coil.compose.rememberAsyncImagePainter
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun DetailScreen() {
+    val localDensity = LocalDensity.current
+    val sellBuyLayoutHeight = remember { mutableStateOf(Dp0) }
+
+    Surface {
+        ConstraintLayout(
+            modifier = Modifier
+                .fillMaxSize()
+                .systemBarsPadding()
+                .verticalScroll(state = rememberScrollState())
+        ) {
+
+            val (
+                appBar,
+                logo,
+                currentPrice,
+                priceChangePercent,
+                graph,
+                coinInfoItem
+            ) = createRefs()
+
+            Appbar(modifier = Modifier
+                .constrainAs(appBar) {
+                    top.linkTo(parent.top)
+                    start.linkTo(parent.start)
+                }
+            )
+
+            Image(
+                modifier = Modifier
+                    .size(Dp60)
+                    .constrainAs(logo) {
+                        top.linkTo(appBar.bottom)
+                        linkTo(start = parent.start, end = parent.end)
+                    }
+                    .padding(top = Dp8),
+                // TODO: Remove dummy image when work on Integrate.
+                painter = rememberAsyncImagePainter(
+                    "https://assets.coingecko.com/coins/images/1/large/bitcoin.png?1547033579"
+                ),
+                contentDescription = null
+            )
+
+            Text(
+                modifier = Modifier
+                    .constrainAs(currentPrice) {
+                        top.linkTo(logo.bottom)
+                        linkTo(start = parent.start, end = parent.end)
+                    }
+                    .padding(vertical = Dp8),
+                // TODO: Remove dummy value when work on Integrate.
+                text = stringResource(R.string.coin_currency, "3,260.62"),
+                color = MaterialTheme.colors.textColor,
+                style = Style.semiBold24()
+            )
+
+            PriceChangeButton(
+                modifier = Modifier
+                    .constrainAs(priceChangePercent) {
+                        top.linkTo(currentPrice.bottom)
+                        linkTo(start = parent.start, end = parent.end)
+                    },
+                iconPaddingEnd = Dp9,
```

Comment:
I think we can replace this param as `displayForDetailPage: Boolean = false`, same to what we did in `PriceChange` 🤔